corehq/apps/app_manager/urls.py

```diff
@@ -128,6 +130,7 @@
     url(r'^download/(?P<app_id>[\w-]+)/',
         include('corehq.apps.app_manager.download_urls')),
     url(r'^formdefs/(?P<app_id>[\w-]+)/', 'formdefs', name='formdefs'),
+    url(r'^ng_template/(?P<template>[\w-]+)', DynamicTemplateView.as_view(), name='ng_template'),
```

Comment:
Used this trick so that I could make use of the Django template engine for the Angular template partials. Of particular use were 'trans' and base templates.